Content.Shared/Verbs/VerbCategories.cs

```diff
@@ -10,7 +10,8 @@ public static class VerbCategories
             ("Debug", "/Textures/Interface/VerbIcons/debug.svg.192dpi.png");
 
         public static readonly VerbCategoryData Rotate = ("Rotate", null);
-        public static readonly VerbCategoryData Construction = ("Construction", null);
+        public static readonly VerbCategoryData Construction =
+            ("Construction", "/Textures/Interface/hammer.svg.192dpi.png");
```

Comment:
This is the same icon as is used for the menu in the upper left of the screen. It works, but is maybe ever so slightly too large for the right click context menu?

Reply:
ask in #art-and-ui or ask swept to resize them, i haven't seen ingame or anything but it'd be better if they were resized properly for it

Reply:
Will do. For reference, it currently looks like this
![icons](https://user-images.githubusercontent.com/60421075/130182125-a661c3ef-5ca0-4dd1-b671-476c32c75ea2.png)

Reply:
I just ended up resizing it myself, and adding a new 'pick up' arrow. Construction **verb** icon is now slightly smaller and actually fit's into its region
![construction](https://user-images.githubusercontent.com/60421075/130207450-0d0f14c7-fb4c-49c7-872b-fd945dd6df01.png)